beetsplug/chroma.py

```diff
@@ -88,16 +113,24 @@ def acoustid_match(log, path):
         return None
     _acoustids[path] = result['id']
 
-    # Get recording and releases from the result.
+    # Get recording and releases from the result,
+    #  sorted by date and/or country depending on 'match'>'preferred' config.
     if not result.get('recordings'):
         log.debug(u'no recordings found')
         return None
     recording_ids = []
-    release_ids = []
+    releases = []
     for recording in result['recordings']:
         recording_ids.append(recording['id'])
         if 'releases' in recording:
-            release_ids += [rel['id'] for rel in recording['releases']]
+            releases.extend(recording['releases'])
+    country_patterns = config['match']['preferred']['countries'].as_str_seq()
```

Comment:
A comment here might be in order to introduce the reason for all this sorting business.